src/styles/blocks/cards/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--#{$stars-quantity} .stars__star:nth-child(-n + #{$stars-quantity}) {
+    background-image: url(/src/images/star-active.svg);
+    background-repeat: no-repeat;
+  }
```

```suggestion
  &--#{$stars-quantity} :nth-child(-n + #{$stars-quantity}) {
    background-image: url(/src/images/star-active.svg);
    background-repeat: no-repeat;
  }
```
I checked, and now if I change modifier stars--4 to stars--3 we don't get the needed result. 
Please fix it, I suggest changes, but you need to check it additionally.

---

I implemented your small improvement, but for organaizing the main idea of this part (automatize with changing "modifier stars--4 to stars--3"), **here we need to use JS (unfortunately, we haven't started learning JS Advanced yet)**, otherwise we have to change _"$stars-quantity"_ in _"variables.scss"_ and _"modifiers stars--N"_ in index.html by self. **Sad, but true**

---

But all works fine. 
Now you can change modifier and get different count of stars (1,2,3 etc.)